src/containers/enrollment/EnrollmentLink.js

```diff
@@ -0,0 +1,42 @@
+import { faCopy } from '@fortawesome/pro-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// @flow
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardSegment,
+} from 'lattice-ui-kit';
+
+import copyToClipboard from '../../utils/copyToClipboard';
+
+const EnrollmentLink = () => {
+  const enrollmentLinkQueryParams = window.location.hash.split('?')[1];
```

Comment:
I don't think this works. Here's an example of a valid link:

```
https://openlattice.com/chronicle/login?organizationId=...&studyId=...&participantId=...
```

There's no hash for this link so `window.location.hash` will return an empty string. Furthermore, to get the query string, you can just use the `search` property:

https://developer.mozilla.org/en-US/docs/Web/API/Location/search

```
const qs = window.location.search;
// qs = "?organizationId=...&studyId=...&participantId=..."
```

Reply:
I understand what you're saying and also initially thought I should use the `search` property to get the query string...but I think I've confused myself as to what I'm trying to accomplish here. Whether logged in or not, navigating to the URL will always add a hash to end of the URL, so the correct page is not rendered. (When logged in, it just redirects to the studies page, which is expected because the AppContainer has a `Redirect` element pointing to that route, and when not logged in, it redirects to `/#/login/`, the actual login page.) Of course, when there is hash included after `/chronicle/`, the page we expect is rendered just fine:

```
https://openlattice.com/chronicle/#/login?organizationId=...&studyId=...&participantId=...

Also... I'm now not sure the query params need to be included at all in the URL. Because I'm not sure it makes sense to even include a button to copy the enrollment link, which is the only place the params are being used. I'm not sure why the user would be copying the link at all, if they've received it via email—I would imagine they'd install the app and then try clicking on the URL again? I could be misunderstanding.
But in that case, I think having the enrollment link route to `https://openlattice.com/chronicle/login/enrollment/` would be sufficient. Is this possible through changes in AWS?